Docker setup

jodal opened the pull request.

jodal committed:
feat: Add Dockerfile

jodal committed:
feat: Add docker-compose with PostgreSQL

This can be useful for quickly getting the app running locally in an
environment close to production.

fuzzy76 commented:
Is there anything in particular holding this up?

jodal commented:
No, not really. comics development happens quite sporadically for me these days, and so far I've been happy with modernizing/getting closer to run it as a container. I still run it the good old way with nginx in front and systemd keeping gunicorn alive.

Do you run a comics instance yourself?

fuzzy76 commented:
I do not, as I use comics.io. But running it through Docker would make it easier for me to contribute new parsers.

jodal committed:
ci: Build and publish the Docker image

After the test matrix passes on a push to main, build and publish the
Docker image to ghcr.io.
